extensions/devservices/oidc/src/main/java/io/quarkus/devservices/oidc/OidcDevServicesConfig.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.devservices.oidc;
+
+import java.util.List;
+import java.util.Map;
+
+import io.quarkus.runtime.annotations.ConfigDocMapKey;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+/**
+ * OpenID Connect Dev Services configuration.
+ */
+@ConfigRoot
+@ConfigMapping(prefix = "quarkus.oidc.devservices")
+public interface OidcDevServicesConfig {
+
+    /**
+     * Use OIDC Dev Services instead of Keycloak.
+     */
+    @WithDefault("false")
```

Comment:
Proposing to keep it Optional, for the processor to assume it is true if no Docker env is available

Reply:
Done.